WcaOnRails/app/views/registrations/_payment_form.html.erb

```diff
@@ -1,278 +1,3 @@
-<% add_to_packs("auto_numeric") %>
-<%= horizontal_simple_form_for :payment, url: "", html: { id: :form_payment } do |f| %>
-  <%= f.input :fees_paid, label: t('registrations.payment_form.labels.fees_paid'), hint: false do %>
-    <p class="form-control-static"><%= format_money(@registration.paid_entry_fees) %></p>
-  <% end %>
-  <%= f.input :fees_remaining, label: t('registrations.payment_form.labels.fees_remaining'), hint: false do %>
-    <p class="form-control-static"><%= format_money(@registration.outstanding_entry_fees) %></p>
-  <% end %>
+<%= render "stripe_payment_form" if @competition.stripe_connected? %>
+<%= render "paypal_payment_form" if @competition.paypal_connected? && !EnvConfig.WCA_LIVE_SITE? && !@competition.stripe_connected? %>
```

Comment:
1. Double-check, see other comment.
2. The additional ` && !@competition.stripe_connected` can be reprhased much more succintly by wrapping this rendering logic in a big if-elsif. `if @competition.stripe_connected render_stripe elsif @competition.paypal_connected? render_paypal`